pkg/gofr/container/container.go

```diff
@@ -75,8 +75,13 @@ func (c *Container) Create(conf config.Config) {
 	}
 
 	if c.Logger == nil {
+		levelFetchConfig, err := strconv.Atoi(conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
+		if err != nil {
+			levelFetchConfig = 15
```

Comment:
No error log message here to report the value cannot be used

Reply:
Gave an error message for incorrect value